msteams-platform/tabs/how-to/app-caching.md

```diff
@@ -143,53 +167,55 @@ Monitor the `isBackgroundLoad` property in the app context to optimize the app f
 
-* The Teams client invokes the `loadHandler` only after the `unload` sequence of the app is completed. For example, if a user launches tab A of your app and then launches tab B of the same app, tab B won't get the load signal until the tab A invokes the `readyToUnload` callback.
+* The suspended app can be used within the same window. The app that is suspended in a pop out window cannot be reused in the Main window.
 
-* Apps are cached on a per-window basis. App caching happens on a per app (not on a per tab) basis within the same window.
+* When an app is suspended, all the registered handlers are deleted. When the app is resumed all the handlers, such as `themeChange` or `focusEnter`, need to be reregistered. No notifications are sent to the app when suspended. If your app requires notifications even when suspended, suspension might not be the right solution.
 
-* App caching isn't supported for the meeting stage or dialog (referred as task module in TeamsJS v1.x) contexts, because these can be opened on top of the tab and the same iframe or webview can't be used to render the content in the tab and the dialog.
+* Only Single-page apps that use client-side routing for page navigation can benefit from app suspension and resume. It's recommended that the same domain be used across all contexts of your app launch.
 
-* Register only the `beforeUnload` handler if your app doesn't require app caching but needs time to safely save state (as leaving the app can cause the app content to be abruptly removed from the Document Object Model (DOM)). If the app hasn't registered for the `load` event, it's removed from the DOM after the `unload` flow completes.
+* An app is expected to sleep when suspended. No SDK requests are allowed when the app is suspended.
+
+## Limitations within Teams
+
+The following are the limitations for app suspension within the Teams app:
+
+* The Teams client invokes the `resume` handler only after the `suspendOrTerminate` sequence of the app is completed. For example, if a user launches tab A of your app and then launches tab B of the same app, tab B won't get the `resume` signal until the `suspendOrTerminate` handler on tab A is done executing.
+
+* Apps are cached on a per-window basis. App caching happens on a per app (not on a per tab) basis within the same window.
 
-* Follow the guidelines in this section to onboard your app to app caching in Teams meeting. For app caching support only in meetings, register the `load` or `beforeUnload` handlers if the context is `sidePanel`.
+* App suspension isn't supported for the meeting stage or dialog (referred as task module in TeamsJS v1.x) contexts, because these can be opened on top of the tab and the same iframe or webview can't be used to render the content in the tab and the dialog.
```

Comment:
The table in the beginning of the doc shows what frameContext Teams support for caching. 
For non-Teams hubs, only `FrameContext.Content` will be cached. That means, `FrameContext.Task` that is inside Dialog in not supported.

Reply:
@lakhveerk Added the below entry with the following text, could you let me know if it looks ok?

The table in the introductory section, [App suspension for your tab app](#app-suspension-for-your-tab-app), provides information about what `frameContext` Teams supports for caching. For non-Teams hubs, only `FrameContext.Content` is cached. That means, `FrameContext.Task`, which is inside `Dialog` is not supported.